youcan-shop/youcan-ui
# YSHOP2-1237: Added new icons

## JIRA Ticket

[YSHOP2-1237](https://youcanshop.atlassian.net/browse/YSHOP2-1237)

##QA steps

- Go to App.vue
- Check the new 7 icons

## Note

Leave empty when you have nothing to say


[YSHOP2-1237]: https://youcanshop.atlassian.net/browse/YSHOP2-1237?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
